cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,52 @@
 /// <reference types="cypress" />
 /// <reference types="../support" />
 
+import SignInPageObject from '../support/pages/signIn.pageObject';
+import HomePageObject from '../support/pages/home.pageObject';
+import SettingsPageObject from '../support/pages/settings.pageObject';
+
+const signInPage = new SignInPageObject();
+const homePage = new HomePageObject();
+const settingsPage = new SettingsPageObject();
+
 describe('Settings page', () => {
-  before(() => {
+  let user;
 
+  before(() => {
+    cy.task('generateUser').then((generatedUser) => {
+      user = generatedUser;
+    });
   });
 
   beforeEach(() => {
-
+    cy.task('db:clear');
+    cy.login();
```

Comment:
Consider adding `data-cy` attributes to elements you interact with in your tests, as per the task requirements. This will make your tests more robust and less prone to breaking due to changes in the UI.